root_VS2010/programs/VB/z_Common2.bat

```diff
@@ -5,18 +5,18 @@
 @rem --------------------------------------------------
 @rem Set Program Files path
 @rem --------------------------------------------------
-reg Query "HKLM\Hardware\Description\System\CentralProcessor\0" | find /i "x86" > NUL && set PROGRAM_FILES=Program Files || set PROGRAM_FILES=Program Files (x86)
+reg Query "HKLM\Hardware\Description\System\CentralProcessor\0" | find /i "x86" > NUL && set PROGRAM_FILES=C:\Program Files || set PROGRAM_FILES=C:\Program Files (x86)
```

Comment:
You should modify this code as follows.

```bat
...... set PROGRAM_FILES=C:\Program Files||set PROGRAM_FILES=C:\Program Files (x86)
```

# Reason
The space character exists at the following position.
```bat
...... set PROGRAM_FILES=C:\Program Files || set PROGRAM_FILES=C:\Program Files (x86)
                                         ^
```

Therefore, the file path of the build tool is as follows.
This file path does not exist.
```bat
C:\Program Files \Microsoft Visual Studio 10.0\Common7\IDE\devenv.com
                ^
```